services/jenkins/jenkins-base.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const { BaseJsonService } = require('..')
+const serverSecrets = require('../../lib/server-secrets')
+
+module.exports = class JenkinsBase extends BaseJsonService {
+  async fetch({
+    url,
+    schema,
+    qs,
+    errorMessages = { 404: 'instance or job not found' },
+  }) {
+    const options = { qs, strictSSL: false }
```

Comment:
I added the `strictSSL` setting while debuging https://github.com/badges/shields/issues/1956. This does technically fix the issue in #1956 but I'm not convinced this is necessarily the path we want to go. 

Curious what others think

Reply:
In terms of shields.io itself, I think we should ideally fix this by upgrading node, but I realise this is currently problematic for...reasons. We could do this as a workaround while that is problematic.

For self-hosting users who want to use this option to deal with self-signed certs, I think we should make this a configurable option rather than turn `strictSSL` off by default

Reply:
> In terms of shields.io itself, I think we should ideally fix this by upgrading node,

This issue still persists on Node 10.x btw, so I think this may be a longer term issue. I through this idea out there in #1956 but I think I'd like to add a query param to these Jenkins badges that folks could use in these exceptional circumstances. I'll only include the `strictSSL` option here when the query param was set. 

Still not ideal IMO but if we end up going down to give Shields users a work-around when there's fix cert chain issues (note this is also happening for Coverity refs #3334) then that would at least keep it pretty narrow.

Thoughts?

Reply:
OK yes agreed. If we make `strictSSL` on by default but all the user to disable with a URL param I think that is a sensible default/viable workaround :+1:

Reply:
Done